app/kvstore/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+//
+// Example application for MegaBlocks implementing a KV store following the user guide
+// of CometBFT (https://docs.cometbft.com/v0.38/guides/go) for applications running as
+// separate process from CometBFT
+//
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	abciserver "github.com/cometbft/cometbft/abci/server"
+	cmtlog "github.com/cometbft/cometbft/libs/log"
+	"github.com/dgraph-io/badger/v3"
```

Comment:
Any reason to not use v4?

I didn't know about badger until I read this PR.

(just asking)